lib/mars-photos.js

```diff
@@ -33,7 +33,9 @@ function marsPhotos() {
     var validCameras = {
       curiosity: ['fhaz', 'rhaz', 'mast', 'chemcam', 'mahli', 'mardi', 'navcam'],
       opportunity: ['fhaz', 'rhaz', 'navcam', 'pancam', 'minites'],
-      spirit: ['fhaz', 'rhaz', 'navcam', 'pancam', 'minites']
+      spirit: ['fhaz', 'rhaz', 'navcam', 'pancam', 'minites'],
+      // eslint-disable-next-line max-len
+      perseverance: ['edl_rucam', 'edl_rdcam', 'edl_ddcam', 'edl_pucam1', 'edl_pucam2', 'navcam_left', 'navcam_right', 'mcz_right', 'mcz_left', 'front_hazcam_left_a', 'front_hazcam_right_a', 'rear_hazcam_left', 'rear_hazcam_right']
```

Comment:
instead of disabling the linter, you can write this on multiple lines to make the linter happy.
```
perserverance: [
  'edl_rucam',
  'edl_rdcam',
  ...
]
```